### PR TITLE
ref(swagger): change the version version of wfm API and delete latest release endpoint

### PR DESCRIPTION
**Pull request by kmala**
the /versions/{train}/{component}/latest endpoint is deleted because there is another endpoint /versions/{train}/{component}/{release} which is making the server ambiguous.

### CODE REVIEW

File: api/swagger-spec/swagger.yml [hunk 1]
```diff
@@ -65,22 +65,6 @@ paths:
           description: unexpected error
           schema:
             $ref: "#/definitions/error"
-  /versions/{train}/{component}/latest:
```
Comment (mboersma) [inline api/swagger-spec/swagger.yml]: Why is this section removed? If that's not strictly related to `ref(swagger): change the version to v3 from v2` could you make it a separate PR or at least a separate commit?

**Comment by mboersma** [comment]
If these commits aren't strictly related to each other, would you mind making them separate PRs? Also the commit message needs a header, like `ref(swagger): delete the latest release endpoint`.

**Comment by kmala** [comment]
these commits are related in that both are doing changes to the wfm api specification.May be if i change the PR to a different name it should be be clear.
